### PR TITLE
Homework2: Ivan Nikitsenka

### PR DESCRIPTION
**Pull request by vanich-nikita**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle)
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks
- [ ] my PR made from the fork feature branch to maxim homework# branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: src/homework2/task1.py [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
    за L товаров.
 """
 
 
 def total_sum(m, n, s):
+
     """Подсчет общей суммы покупок.
 
     :param m: рубли
+
```
Comment (belovmd) [inline src/homework2/task1.py]: redundant empty lines

File: src/homework2/task1.py [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
     :param n: копейки
+
```
Comment (belovmd) [inline src/homework2/task1.py]: looks too bulky

File: src/homework2/task7.py [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+XO("ooxXm") => true
+XO("zpzpzpp") => true // when no 'x' and 'o' is present should return true
+XO("zzoo") => false
+'''
+
+s = input("Input string:")
+s = s.lower()
+o, x = 0, 0
+flag = False
+for i in s:
```
Comment (belovmd) [inline src/homework2/task7.py]: str.count?

File: src/homework2/task7.py [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+is_isogram("Dermatoglyphics" ) == true
+is_isogram("aba" ) == false
+is_isogram("moOse" ) == false # -- ignore letter case
+'''
+
+string = input("Input string: ")
+flag = True
+string = string.lower()
+for i in string:
```
Comment (belovmd) [inline src/homework2/task7.py]: o(n**2) complexity. 
has O(n) using sets

**Review by belovmd — COMMENTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-summer/blob/master/Homework%20review/Homework2.md):
- 10
- 16

- two files with similar tasks!

File: src/homework2/task1.py (hunk 1 above)
Comment (vanich-nikita) [reply src/homework2/task1.py]: fixed

File: src/homework2/task1.py (hunk 2 above)
Comment (vanich-nikita) [reply src/homework2/task1.py]: fixed

File: src/homework2/task7.py (hunk 3 above)
Comment (vanich-nikita) [reply src/homework2/task7.py]: done

File: src/homework2/task7.py (hunk 4 above)
Comment (vanich-nikita) [reply src/homework2/task7.py]: done

**Review by belovmd — COMMENTED** [review]
weird comments (task 1, 5) - checklist item, previous review note. 
8,5

**Pull request closed without merging**